cmd/av/stack_next.go

```diff
@@ -39,6 +30,39 @@ child branch (without recursively syncing further descendants).
 			return errors.New("invalid number (must be >= 1)")
 		}
 
-		return errors.New("unimplemented")
+		// Get the subsequent branches so we can checkout the nth one
+		repo, _, err := getRepoInfo()
+		if err != nil {
+			return err
+		}
+		branches, err := meta.ReadAllBranches(repo)
+		if err != nil {
+			return err
+		}
+		currentBranch, err := repo.CurrentBranchName()
+		if err != nil {
+			return err
+		}
+		subsequentBranches, err := subsequentBranches(branches, currentBranch)
+		if err != nil {
+			return err
+		}
+
+		// confirm we can in fact do the operation given current branch state
+		if subsequentBranches == nil {
+			return errors.New("there is no next branch")
+		}
+		if n > len(subsequentBranches) {
+			return fmt.Errorf("invalid number (there are only %d subsequent branches in the stack)", len(subsequentBranches))
+		}
+
+		// checkout nth branch
+		if _, err := repo.CheckoutBranch(&git.CheckoutBranch{
+			Name: subsequentBranches[n-1],
+		}); err != nil {
+			return err
+		}
+
```

Comment:
Lets add some output that says what branch they're on.

Something like `Checkout out branch <my branch>` (use the UserInput color thing for the branch name, and print to stderr).